### DISCUSSION TITLE
How can designers create Carbon Design System page content?

### DISCUSSION THREAD

**Discussion by ssdesign** [carbon-design-system/carbon#15171] [Accessibility]
Hi,
I was wondering how do designers create Carbon design system pages?
Do they always have to rely on developers to build these pages for them? or is there an "Edit" mode that can allow anyone to add content once I host Carbon on my site.

Any advice or help is appreciated.

Thanks

**Comment by John30013** [comment]
Yes, you'll need a web developer to create pages using the Carbon Design System. There's no "drag-and-drop" page builder (or similar) for it. The Carbon Design System can be used with plain HTML pages, but it is designed in React, and the other supported frameworks (plain HTML, Angular, Svelt and Vue) are community-supported.
